src/app/core-ui/main/status/modal/help-modal/console-modal.component.ts

```diff
@@ -42,17 +46,11 @@ export class ConsoleModalComponent implements OnInit, AfterViewChecked {
   }
 
   rpcCall() {
-    const params = this.command.split(' ');
-
-    // TODO: Remove next release
-    const daemonVersion = this._rpc.state.get('subversion');
```

Comment:
with which daemons will that work / not work ?

Reply:
it will work for latest daemon 0.15.1.2